src/mtype.d

```diff
@@ -8323,7 +8323,24 @@ public:
             {
                 return getProperty(e.loc, ident, flag);
             }
-            return sym.getMemtype(Loc()).dotExp(sc, e, ident, flag);
+
+            if (flag == 0)
+                flag = 1;  // silence lookup errors for memtype
+
+            Expression res = sym.getMemtype(Loc()).dotExp(sc, e, ident, flag);
+            if (!res)
+            {
+                if (auto ns = sym.search_correct(ident))
+                    e.error("no property '%s' for type '%s'. Did you mean "
+                        "'%s.%s' ?", ident.toChars(), toChars(), toChars(),
```

Comment:
It's not a hard limit, and it's not followed very often, and magicport removed all the manual line breaks.
Plus, implicit string literal concatenation is evil.